README.md

```diff
@@ -64,6 +64,25 @@ The generation task currently creates the following signatures:
 - Named scopes
 - Model relation class
 
+### Overriding generated sigs
+
+`sorbet-rails` rely on Rails reflection to generate the signature. There are features we haven't supported yet such as [serialize](https://github.com/chanzuckerberg/sorbet-rails/issues/49) and [attribute custom type](https://github.com/chanzuckerberg/sorbet-rails/issues/16). We don't know the sig of a method if you have overridden it as well. However, it is possible to override the sigs that `sorbet-rails` generated. 
+
+For example, here is how to override sig for a method in a model.
+
+```ruby
+# in app/models/<model_name>.rbi
```

Comment:
I'd suggest not using `<` and `>` because in code below it is unclear whether it is part of the syntax or not.

```suggestion
# in app/models/model_name.rbi
```